src/cookbooks/additional_props.md

```diff
@@ -0,0 +1,50 @@
+# Additional Properties
+
+If some field is present in a valid received [ping](../concepts/terminology.md#ping),
+but is not present in the ping's [schema](../concepts/terminology.md#schema),
+it doesn't have its own column to be placed into during
+[ingestion](../concepts/terminology.md#ingestion).
+Instead, those fields remain as raw JSON and are placed in the
+`additional_properties` column of the ping's table or view.
+
+This can happen for a variety of usually-temporary problems like:
+
+- The latest schema hasn't yet been deployed (see ["What does it mean when a schema deploy is blocked?"](../concepts/pipeline/schemas.html#what-does-it-mean-when-a-schema-deploy-is-blocked))
```

Comment:
It is important to note that we do not backfill schema changes. So when the new column gets added, new data will flow there, and the existing query will be wrong.